include/aie/Dialect/AIEX/IR/AIEX.td

```diff
@@ -868,6 +869,18 @@ def AIE_DMAConfigureTaskOp : AIEX_Op<"dma_configure_task", [HasParent<"RuntimeSe
   let hasCanonicalizeMethod = 1;
 }
 
+def AIE_DMAConfigureTaskForOp : AIEX_Op<"dma_configure_task_for", [HasParent<"RuntimeSequenceOp">]>, Results<(outs Index:$result)> {
```

Comment:
Can you add a summary documenting this OP?

Reply:
Added